hack/update-codegen.sh

```diff
@@ -22,8 +22,16 @@ source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/library.sh
 
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${REPO_ROOT_DIR}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../../../k8s.io/code-generator)}
 
+go install ./vendor/k8s.io/code-generator/cmd/deepcopy-gen
```

Comment:
you should not have to do this, the test-infra code handles installing and running in vendor.